### DISCUSSION TITLE
Minimum supported meson revision?

### DISCUSSION THREAD

**Discussion by NJRoadfan** [Netatalk/netatalk#958] [Q&A]
OS: Debian 11 Bullseye.

Tried to give the new meson build system a spin on a test VM running Debian 11 (which still includes an OpenSSL release that supports DHX). Seems that the included version of meson is too old (0.56.2) and is requiring at least version 0.61.2. Realistically, what is the oldest supported version? I tried forcing a build with 0.56.2, but it is indeed too old, needing `global_build_root()` which was introduced in 0.58.

**Comment by rdmark** [comment]
I think 0.61.2 was chosen for good reasons...

There's a good chance you can get a newer version of meson on Bullseye via python pip. Try uninstalling the meson deb, and then install the python pip package with the same name, e.g. as a system package.

```
sudo pip3 install meson
```

**Reply by rdmark** [reply]
@NJRoadfan Did you have a chance to try the pip method on your system?
